Source/Scene/Camera.js

```diff
@@ -29,6 +29,30 @@ import CameraFlightPath from "./CameraFlightPath.js";
 import MapMode2D from "./MapMode2D.js";
 import SceneMode from "./SceneMode.js";
 
+//PROPELLER HACK
+function clampSphere(sphere, position, result) {
+  if (
+    !defined(sphere) ||
+    !defined(position) ||
+    !position ||
+    position === Cartesian3.ZERO
```

Comment:
Is this what you want?  Triple equals is like comparing pointers.  It literally has to be `Cartesian3.ZERO` not a `new Cartesian3(0,0,0)`

![image](https://user-images.githubusercontent.com/484870/90587692-5b2fa700-e21d-11ea-8324-b3fcd70ac1ae.png)

If you want to compare the value you can use `Cartesian3.equals(position, Cartesian3.ZERO)`

Reply:
Oh right